src/repository/index.ts

```diff
@@ -77,6 +78,8 @@ export default class Repository extends EventEmitter implements EventEmitter {
 
   private data: FeatureToggleData = {};
 
+  private knownSegments?: Segment[];
```

Comment:
Could this be `segments`? Not sure what `known` implies.

Reply:
It could be segments! So the known is there because this is not all the segments but only the ones that are actively used. But the node sdk doesn't actually really know that or even really care about that. So... good point, I'll change this to segments